web/packages/teleport/src/Assist/LandingPage.tsx

```diff
@@ -0,0 +1,180 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useCallback } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+import {
+  AuditLogIcon,
+  PlusIcon,
+  RemoteCommandIcon,
+  SearchIcon,
+  ServerIcon,
+} from 'design/SVGIcon';
+
+import { useHistory } from 'react-router';
+
+import Flex from 'design/Flex';
+
+import cfg from 'teleport/config';
+import { useConversations } from 'teleport/Assist/contexts/conversations';
+
+const Container = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: center;
+`;
+
+const Content = styled.div`
+  background: ${p => p.theme.colors.levels.popout};
+  color: ${p => p.theme.colors.text.main};
+  font-size: 15px;
+  padding: 20px 25px;
+  border-radius: 7px;
+  margin-top: 50px;
+  width: 700px;
+`;
+
+const Title = styled.h2`
+  margin: 0 0 20px;
+`;
+
+const SubTitle = styled.h3`
+  margin: 0 0 10px;
+`;
+
+const Features = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+`;
+
+const Feature = styled.div`
+  background: ${p => p.theme.colors.spotBackground[0]};
+  margin-bottom: 10px;
+  margin-right: 20px;
+  padding: 10px 15px;
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+  font-size: 15px;
+
+  svg {
+    margin-right: 15px;
+
+    path {
+      fill: ${p => p.theme.colors.text.main};
+    }
+  }
+`;
+
+const Warning = styled.div`
+  border: 2px solid ${p => p.theme.colors.warning.main};
+  border-radius: 7px;
+  padding: 10px 15px;
+  margin-bottom: 15px;
+`;
+
+const NewChatButton = styled.div`
+  padding: 10px 20px;
+  border-radius: 7px;
+  font-size: 15px;
+  font-weight: bold;
+  display: flex;
+  cursor: pointer;
+  margin: 0 15px;
+  background: ${p => p.theme.colors.buttons.primary.default};
+  color: ${p => p.theme.colors.buttons.primary.text};
+  align-items: center;
+  justify-content: center;
+
+  svg {
+    position: relative;
+    margin-right: 10px;
+  }
+
+  &:hover {
+    background: ${p => p.theme.colors.buttons.primary.hover};
+  }
+`;
+
+export function LandingPage() {
+  const theme = useTheme();
+
+  const history = useHistory();
+
+  const { create } = useConversations();
+
+  const handleNewChat = useCallback(() => {
+    create().then(conversationId =>
+      history.push(cfg.getAssistConversationUrl(conversationId))
+    );
+  }, []);
+
+  return (
+    <Container>
+      <Content>
+        <Title>Teleport Assist</Title>
+
+        <p>
+          Teleport Assist utilizes facts about your infrastructure to help
+          answer questions, generate command line scripts and help you perform
+          routine tasks on target resources.
+        </p>
+
+        <Warning>
```

Comment:
Legal has specific language they want to see, so should work with them on final language here.

Reply:
@klizhentas Legal does have some requirements. I'll follow-up with you separately.